src/coord/src/catalog.rs

```diff
@@ -813,7 +815,7 @@ impl Catalog {
     /// This function should not be called in production contexts. Use
     /// [`Catalog::open`] with appropriately set configuration parameters
     /// instead.
-    pub fn open_debug(path: &Path) -> Result<Catalog, anyhow::Error> {
+    pub fn open_debug(path: &Path, now: NowFn) -> Result<Catalog, anyhow::Error> {
```

Comment:
Every caller of this passes `ore::now::now_zero`—can we push that into the function itself? (The idea of this function is to basically no parameters and just "do the right thing" for tests.)

Reply:
I want it to be an argument because of https://github.com/MaterializeInc/materialize/pull/7144 which will use a non-zero fn.

Reply:
Ack.